jobs/service-fabrik-serviceflow-manager/monit

```diff
@@ -0,0 +1,5 @@
+check process service-fabrik-serviceflow-manager with pidfile /var/vcap/sys/run/service-fabrik-serviceflow-manager/service-fabrik-serviceflow-manager.pid
```

Comment:
Can we rename job to `service-fabrik-serviceflow-operator`. I couldn't rename existing manager jobs because it would have removed old logger files

Reply:
@amitmalav I did initially name it operator but then i kept it inline with other jobs as i wanted to keep it consistent. Whenever this refactoring exercise would happen at that time this new job can be additionally refactored. I really do not like codebase to have multiple confusing nomenclatures.

Reply:
@amitmalav @sauravmndl @kamath-prasad I have gone ahead and included just the min props required for the job. Going fwd we can set this as the guideline for any job. Also, we can add a backlog to clear existing settings from other jobs.